defaults/main.yml

```diff
@@ -62,7 +62,7 @@ sensu_api_port: 4567
 sensu_api_ssl: "false"
 sensu_api_user_name: admin
 sensu_api_password: secret
-sensu_api_uchiwa_path: ''
+sensu_api_sensu_uchiwa_path: ''
```

Comment:
double sensu here? (I assume you just used `sed` and caught this twice)

Reply:
Lol...  Oops?   Thought I caught all the doubles :(

Reply:
Haha all good!